brainstormforce/ultimate-addons-for-gutenberg
# Fix: Lottie - SVG broken in editor.

### Description

- See here - https://share.getcloudapp.com/OAu4yNr7

### Screenshots

- Output - https://share.getcloudapp.com/xQuz5QLD

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Lottie in next-release and check Lottie block SVG on inspector setting, sidebar, and UAG blocks in right side.
- Switch to the above branch and check the same it will fix the issue.

### Checklist:
